webapp/src/components/modals/attach_comment_to_issue/attach_comment_to_issue.jsx

```diff
@@ -58,7 +59,7 @@ export default class AttachIssueModal extends PureComponent {
 
         this.props.create(issue).then((created) => {
             if (created.error) {
-                let errMessage = created.error.message;
+                let errMessage = getErrorMessage(created.error.message);
                 if (created.error.response &&
                     created.error.response.body &&
                     created.error.response.body.message) {
```

Comment:
I think the whole `if (created.error.response` if block can be removed here, now that the error message is being parsed beforehand.
```suggestion
                const errMessage = getErrorMessage(created.error.message);
```